reinitialize-pods/.gitignore

```diff
@@ -0,0 +1 @@
+/target
```

Comment:
hmm do we need this? won't this compile in the workspace's target dir? e.g. the proxy init root.

Reply:
Good catch, just a left-over from when I had this in a separate repo on its own :-)